webroot/css/scss/layout/_nav.scss

```diff
@@ -340,6 +342,8 @@ nav {
                 justify-content: center;
                 padding: 2rem;
```

Comment:
can we use vars for spacing and unify px/rem/em?

Reply:
Of course!!